src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -199,7 +199,7 @@ class Request
     protected $defaultLocale = 'en';
 
     /**
-     * @var array
+     * @var array<string, array<string>>
```

Comment:
`string[]` here also, or `list<string>` (but I prefer the former)

Reply:
There are the code 
```
static::$formats[$format] = \is_array($mimeTypes) ? $mimeTypes : [$mimeTypes];
```
so it's an array of array.

Did you mean you preferred `array<string, string[]>` over `array<string, array<string>>` ?

Reply:
> Did you mean you preferred array<string, string[]> over

yes :)

Reply:
Done :)